udata/api/errors.py

```diff
@@ -0,0 +1,6 @@
+from udata.i18n import gettext as _
```

Comment:
You should be using lazy_gettext in order to have the language resolved on query time instead of declaration time.